submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/ResubmissionApiTest.kt

```diff
@@ -383,4 +394,82 @@ class ResubmissionApiTest(
             .isThrownBy { webClient.submitSingle(version2, TSV) }
             .withMessageContaining("The release date of a public study cannot be changed")
     }
+
+    @Test
+    fun `5-7 not authorized user updates public submission files`() =
+        runTest {
+            val version1 =
+                tsv {
+                    line("Submission", "S-RSTST8")
+                    line("Title", "Update Submission Files")
+                    line("ReleaseDate", OffsetDateTime.now().toStringDate())
+                    line()
+                    line("Study")
+                    line()
+                    line("File", "file_5-7-1.txt")
+                    line()
+                    line("File", "file_5-7-2.txt")
+                    line()
+                }.toString()
+
+            webClient.uploadFile(tempFolder.createFile("file_5-7-1.txt", "5-7-1 file content"))
+            webClient.uploadFile(tempFolder.createFile("file_5-7-2.txt", "5-7-2 file content"))
+            assertThat(webClient.submitSingle(version1, TSV)).isSuccessful()
+
+            val version2 =
+                tsv {
+                    line("Submission", "S-RSTST8")
+                    line("Title", "Update Submission Files")
+                    line("ReleaseDate", OffsetDateTime.now().toStringDate())
+                    line()
+                    line("Study")
+                    line()
+                    line("File", "file_5-7-1.txt")
+                    line()
+                }.toString()
+
+            webClient.uploadFile(tempFolder.createOrReplaceFile("file_5-7-1.txt", "5-7-1 file updated content"))
+            webClient.submitAsync(version2, TSV)
+
+            waitUntil(timeout = TWO_SECONDS) { requestRepository.getRequestStatus("S-RSTST8", 2) == INVALID }
+        }
+
+    @Test
+    fun `5-8 authorized user updates public submission files`() =
```

Comment:
I think we need  3  more scenarios. 
1. Allow pagetab files to be modified (without specific permisions).
2. Allow file list metadata to be modified (without specific permisions).
3. File list files can not be modified

Reply:
Number 1 is [already there](https://github.com/EBIBioStudies/biostudies-backend-services/blob/master/submission/submission-webapp/src/itest/kotlin/ac/uk/ebi/biostd/itest/test/submission/submit/ResubmissionApiTest.kt#L312)

Number 2 and 3 were added to the latest commit